packages/core/src/external/aws/secret-manager.ts

```diff
@@ -0,0 +1,15 @@
+import * as AWS from "aws-sdk";
+
+export function makeSecretManagerClient(region: string) {
+  return new AWS.SecretsManager({ region });
+}
+
+export async function getCodeFromSecretManager(
```

Comment:
"code" is specific, whereas this function returns any secret value... maybe name it `getSecretValue()`?